book/book.toml

```diff
@@ -0,0 +1,3 @@
+title = "Amethyst Documentation"
+author = "Amethyst Developers"
+dest = "html"
```

Comment:
If the destination is set to `html` here, why is `.gitignore` also being changed to ignore `book/book` instead of `book/html`?